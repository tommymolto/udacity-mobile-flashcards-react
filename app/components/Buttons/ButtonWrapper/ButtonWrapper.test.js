/* global describe, it, expect */

import React from 'react';
import renderer from 'react-test-renderer';
import ButtonWrapper from './ButtonWrapper';

describe('ButtonWrapper', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ButtonWrapper />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
