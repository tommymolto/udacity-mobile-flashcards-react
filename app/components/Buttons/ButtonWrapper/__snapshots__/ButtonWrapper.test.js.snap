// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonWrapper renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "marginVertical": 50,
        "paddingBottom": 25,
        "paddingHorizontal": 25,
        "paddingTop": 20,
      },
      undefined,
    ]
  }
/>
`;
